tslint-server/src/server.ts

```diff
@@ -109,12 +109,15 @@ folder using \'npm install tslint\' or \'npm install -g tslint\' and then press
 // Options passed to tslint
 let options: tslint.ILinterOptions = {
 	formatter: "json",
-	configuration: {},
+	fix: false,
```

Comment:
You should get the option configuration that is passed in: `tslint.autoFixOnSave`